src/Compilers/Core/MSBuildTask/Microsoft.CSharp.Core.targets

```diff
@@ -198,5 +198,7 @@
     <CommandLineArgsForDesignTimeEvaluation>-langversion:$(LangVersion)</CommandLineArgsForDesignTimeEvaluation>
     <CommandLineArgsForDesignTimeEvaluation Condition="'$(ChecksumAlgorithm)' != ''">$(CommandLineArgsForDesignTimeEvaluation) -checksumalgorithm:$(ChecksumAlgorithm)</CommandLineArgsForDesignTimeEvaluation>
     <CommandLineArgsForDesignTimeEvaluation Condition="'$(DefineConstants)' != ''">$(CommandLineArgsForDesignTimeEvaluation) -define:$(DefineConstants)</CommandLineArgsForDesignTimeEvaluation>
+    <CommandLineArgsForDesignTimeEvaluation Condition="'$(Features)' != ''">$(CommandLineArgsForDesignTimeEvaluation) -features:$(Features)</CommandLineArgsForDesignTimeEvaluation>
+    <CommandLineArgsForDesignTimeEvaluation Condition="'$(DocumentationFile)' != ''">$(CommandLineArgsForDesignTimeEvaluation) -doc:"$(DocumentationFile)"</CommandLineArgsForDesignTimeEvaluation>
```

Comment:
I'm not sure how to review this change from the compiler's perspective. Ideally, I would like to see tests with different values (e.g., to verify escaping works). But it seems this is really an IDE only property, not used by the compiler at all, right?

Reply:
Yeah, I don't really know how this fits into the compiler side of things, AFAIK this is just used as part of what the IDE feeds into the project build. @jasonmalinowski -- do you think this needs compiler side reviews too?

Reply:
Want to keep things simple and say that it's in the compiler code, we need two compiler sign offs. Even if the sign offs are as simple as us validating that "this is not compiler code"